main.go

```diff
@@ -572,17 +572,91 @@ func transformCompile(args []string) ([]string, error) {
+	"hash":                                   true,
+	"hash/crc32":                             true,
+	"internal/bytealg":                       true,
+	"internal/cpu":                           true,
+	"internal/fmtsort":                       true,
+	"internal/nettrace":                      true,
+	"internal/oserror":                       true,
+	"internal/poll":                          true,
+	"internal/race":                          true,
+	"internal/reflectlite":                   true,
+	"internal/singleflight":                  true,
+	"internal/syscall/execenv":               true,
+	"internal/syscall/unix":                  true,
+	"internal/syscall/windows":               true,
+	"internal/syscall/windows/registry":      true,
+	"internal/syscall/windows/sysdll":        true,
+	"internal/testlog":                       true,
+	"internal/unsafeheader":                  true,
+	"io":                                     true,
+	"io/fs":                                  true,
+	"io/ioutil":                              true,
+	"math":                                   true,
+	"math/bits":                              true,
+	"net":                                    true,
+	"os":                                     true,
+	"os/signal":                              true,
+	"path":                                   true,
+	"path/filepath":                          true,
+	"plugin":                                 true,
+	"reflect":                                true,
+	"runtime":                                true,
+	"runtime/cgo":                            true,
+	"runtime/debug":                          true,
+	"runtime/internal/atomic":                true,
+	"runtime/internal/math":                  true,
+	"runtime/internal/sys":                   true,
+	"runtime/metrics":                        true,
+	"runtime/pprof":                          true,
+	"runtime/trace":                          true,
+	"sort":                                   true,
+	"strconv":                                true,
+	"strings":                                true,
+	"sync":                                   true,
+	"sync/atomic":                            true,
+	"syscall":                                true,
+	"syscall/js":                             true,
+	"text/tabwriter":                         true,
+	"time":                                   true,
+	"unicode":                                true,
+	"unicode/utf8":                           true,
+	"unicode/utf16":                          true,
+	"unsafe":                                 true,
+	"vendor/golang.org/x/net/dns/dnsmessage": true,
+}
 
 // isPrivate checks if GOPRIVATE matches path.
 //
 // To allow using garble without GOPRIVATE for standalone main packages, it will
 // default to not matching standard library packages.
 func isPrivate(path string) bool {
-	if module.MatchPrefixPatterns(neverPrivate, path) {
+	if runtimeRelated[path] {
```

Comment:
Doesn't this panic for non existing values?

Reply:
Nope, non existing values return the zero value, i.e. false. Otherwise practically all tests would panic.